Extensions/Today/TodayViewController.swift

```diff
@@ -171,10 +183,12 @@ class TodayViewController: UIViewController, NCWidgetProviding {
     // MARK: Button behaviour
     @objc func onPressNewTab(_ view: UIView) {
         openContainingApp("?private=false")
+//        UnifiedTelemetry.recordEvent(category: .appExtensionAction, method: .press, object: .TodayWidgetNewSearch, value: .normalTab)
```

Comment:
nit: can get rid of commented out code as we no longer need unified telemetry code